Python/initconfig.c

```diff
@@ -2085,6 +2085,11 @@ config_read(PyConfig *config, int compute_path_config)
     /* -X options */
     const wchar_t* option = _Py_check_xoptions(&config->xoptions, known_xoptions);
     if (option != NULL) {
+        if ((wcslen(option) * sizeof(wchar_t)) < 100) {
+            static char error_msg[150];
```

Comment:
This variable will always be allocated even if there is no error, and it's reserved for this exact line of code. I'm not really excited by this implementation.

@pablogsal wanted to do something similar when he wrote the option.

Moreover, PyStatus API uses a bytes string, whereas the option is a wide character string. I expect encoding issues :-(

Reply:
> This variable will always be allocated even if there is no error, and it's reserved for this exact line of code.

It is a static variable to it will be in bss segment.

> Moreover, PyStatus API uses a bytes string, whereas the option is a wide character string. I expect encoding issues :-(

It is just to help the user of typos, it does not need to account for every locale or encoding IMO.

Reply:
@pablogsal Do you have any opinion on this PR?